pkg/container/lima.yaml

```diff
@@ -0,0 +1,30 @@
+# we pretty much take the default of everything, except for specifying the images and the mounts
+
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
```

Comment:
We should generate wolfi-based images for lima, although that will probably require some platform adaptation work.

Reply:
That makes a lot of sense. Is that something we can do here, or is it enough that we can start with this and then iterate?